package.json

```diff
@@ -31,13 +31,16 @@
   },
   "dependencies": {
     "@grpc/proto-loader": "^0.5.0",
+    "@improbable-eng/grpc-web": "^0.9.6",
```

Comment:
Remove this package since it's not used.

Reply:
Done.